camera-core/src/main/java/com/stripe/android/camera/scanui/SimpleScanStateful.kt

```diff
@@ -1,22 +1,22 @@
-package com.stripe.android.stripecardscan.scanui
+package com.stripe.android.camera.scanui
 
 import android.util.Log
 import androidx.annotation.RestrictTo
 import com.stripe.android.camera.framework.AnalyzerLoopErrorListener
-import com.stripe.android.stripecardscan.framework.Config
 
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 abstract class ScanState(val isFinal: Boolean)
 
-internal interface SimpleScanStateful<State : ScanState> {
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+interface SimpleScanStateful<State : ScanState> {
 
-    var scanStatePrevious: ScanState?
-    var scanState: ScanState
+    var scanStatePrevious: State?
+    var scanState: State?
```

Comment:
make the initial scanState nullable - this won't affect stripecardscan, but is needed for identity, where it's implemented by a ViewModel, which doesn't have initial scanState while initialized

Reply:
hmm seems not possible, the compiler says `'lateinit' modifier is not allowed on abstract properties`

Reply:
aha, bummer.